terminology.md

```diff
@@ -247,6 +247,55 @@ In Firefox's Web Console the [command line interpreter](https://developer.mozill
 - [REPL](https://en.wikipedia.org/wiki/Read%E2%80%93eval%E2%80%93print_loop)
 .....
 
+### Agent
+
+#### Definition
+An Agent is a thread that holds all state within a JavaScript VM instance.
+
+### Realm
+#### Definition
+A realm consists of a set of [intrinsic](###Intrinsic) objects, an ECMAScript global environment, all of the ECMAScript code that is loaded within the scope of that global environment, and other associated state and resources (i.e. a global object and an associated set of [primordial](###Primoridal) objects). Objects within a realm implicitly share these primordials and can therefore easily disrupt each other by primordial poisoning -- modifying these objects to behave badly. Today, in the browser, realms can be created via same origin iframes.
```

Comment:
> Objects within a realm implicitly share these primordials and can therefore easily disrupt each other by primordial poisoning -- modifying these objects to behave badly.

This part goes beyond "quick definition".

Reply:
Fixed. Thanks!